src/sentry/snuba/discover.py

```diff
@@ -288,10 +292,10 @@ def query(
         "end": snuba_filter.end,
         "conditions": snuba_filter.conditions,
         "filter_keys": snuba_filter.filter_keys,
+        "having": snuba_filter.having,
```

Comment:
You might need similar logic for `timeseries_query`, and `get_facets` as well.

Reply:
Added.